.github/workflows/build.yml

```diff
@@ -149,6 +149,7 @@ jobs:
         with:
           name: cronos
           extraPullNames: dapp
+          installCommand: "nix-env --quiet -j8 -iA cachix -f https://cachix.org/api/v1/install/1.6.1"
```

Comment:
what's the error without this line?

Reply:
I think they [fix](https://github.com/cachix/cachix-action/issues/168#issuecomment-1900173339) it last friday, can close this now